db/migrate/20190913125420_create_votes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateVotes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :votes, id: :uuid do |t|
+      t.references :user, foreign_key: true, type: 'uuid'
+      t.string :course_slug, null: false
+      t.boolean :liked, null: false, default: false
+
+      t.timestamps
+    end
```

Comment:
To ensure that the count query for the course is fast, need an index:

```ruby
t.index :course_slug
```

Reply:
Done.

Reply:
I don't see this in the current diff?